schema.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE animals (
+  id integer NOT NULL GENERATED ALWAYS AS IDENTITY,
+  name TEXT,
```

Comment:
- [ ] the name should be `varchar()`

Reply:
ok, I understand. Thank you for your kind feedback.